openassessment/xblock/static/js/spec/lms/oa_self.js

```diff
@@ -115,4 +115,15 @@ describe("OpenAssessment.SelfView", function() {
 
         expect(view.baseView.unsavedWarningEnabled()).toBe(false);
     });
+    
+    it("Step should have aria-controls and be visible by default", function() {
```

Comment:
Please create a helper method for the "common" version of this code to reduce duplication.